src/CheckIn.js

```diff
@@ -285,7 +285,7 @@ class CheckIn extends React.Component {
   }
 
   showLoanDetails(loan) {
-    this.props.history.push(`/users/view/${loan.userId}?layer=loan&loan=${loan.id}`);
+    this.props.history.push(`/users/${loan.userId}/loans/view/${loan.id}`);
```

Comment:
This path is used in two places. Let's create a separate function for it to avoid duplication.